modules/monitoring/templates/govuk-uptime-collector.conf.erb

```diff
@@ -0,0 +1,5 @@
+description "govuk-uptime-collector"
+
+start on runlevel [2345]
+
+exec /usr/lib/rbenv/versions/2.3.1/bin/ruby /usr/local/bin/govuk_uptime_collector <%= @environment %> publishing-api content-store
```

Comment:
Do other processes specify ruby version like this? It feels a bit precarious as to when we drop 2.3.1 from stack. Is there anyway we can tell rbenv to run it against a 2.* version of Ruby?